prov/shm/src/smr_attr.c

```diff
@@ -58,7 +58,7 @@ struct fi_rx_attr smr_rx_attr = {
 	.op_flags = SMR_RX_OP_FLAGS,
 	.comp_order = FI_ORDER_STRICT,
 	.msg_order = SMR_RMA_ORDER | FI_ORDER_SAS,
-	.size = 1024,
+	.size = 16384,
```

Comment:
do we know how 16384 is sufficiently enough? and how much more memory this could cost?

Reply:
> do we know how 16384 is sufficiently enough? 

This would depend on application. This value can be overridden by an env var FI_SHM_RX_SIZE.

>  and how much more memory this could cost?

This value is number of `smr_cmd_ctx`. Each `smr_cmd_ctx` is 280 bytes. So set this value to 16384 increase memory usage by ~4.5 MB per endpoint.